mongoose/RestaurantSchema.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file Implements mongoose schema for restaurants
+ */
+import mongoose from "mongoose";
+
+/**
+ * @typedef {Restaurant} Restaurant Represents a restaurant object
+ * @property {string} name Restaurant name
+ * @property {string} bannerPicture Banner picture
+ * @property {string} profilePicture Avatar picture
+ * @property {string} handle Restaurant account handle
+ * @property {string} bio Restaurant bio
+ * @property {string} cuisine Restaurant cuisine type
+ * @property {string} price Restaurant price range
+ * @property {string} address Restaurant address
+ * @property {string} hours Restaurant open hours
```

Comment:
string hours listed here in jsdoc but not in actual schema

Reply:
Thank you for pointing it out! I deleted it from jsdoc.